README.md

```diff
@@ -0,0 +1,3 @@
+## Interview Question Solutions
+
+Solutions to interview questions I faced (created for my own sanity and understanding)
```

Comment:
Change this description too

Reply:
to what?

Reply:
Change it to something else.

Reply:
Oh my gosh, some kind of suggestion would be "helpful" you know!! I changed it by the way.